docs/source/author/writing_philosophy.rst

```diff
@@ -1,2 +1,9 @@
 Writing Philosophy
 ##############################################################################
+
+- how we present material: 
+    - Chapters cover a key area of computer science 
+        - contain an intro page and several chapter sections that each cover a key sub area of the main chapter topic
+        - always end with 'the whole story' section which is a summary of the chapter and a further reading section which contains links/book references 
+          where students can read further about the topic
```

Comment:
Continuation of the same sentence should be on the same line (probably).
Actually this is fairly consistently done throughout so something to make a decision on probably

Reply:
Might be something to put in the writing guide instead of philosophy? Thoughts?

Reply:
I mean in this .rst file (and all the others), not talking about suggestions for content I mean the sentences literally continue on new lines

Reply:
Ohhhhh I see. Yeah it was like that when I copied from CSU docs. I think I'll change it so that they continue on the same line?